book/tut2_spec_pd_gpd.ipynb

```diff
@@ -0,0 +1,961 @@
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>NAME</th>\n",
+       "      <th>Shape_Leng</th>\n",
+       "      <th>geometry</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>Danube</td>\n",
+       "      <td>2.770357e+06</td>\n",
+       "      <td>MULTILINESTRING ((4185683.29 2775788.04, 41861...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>Douro</td>\n",
+       "      <td>8.162452e+05</td>\n",
+       "      <td>MULTILINESTRING ((2764963.81 2199037.624, 2766...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>Ebro</td>\n",
+       "      <td>8.269909e+05</td>\n",
+       "      <td>MULTILINESTRING ((3178371.814 2315100.781, 317...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Elbe</td>\n",
+       "      <td>1.087288e+06</td>\n",
+       "      <td>MULTILINESTRING ((4235352.373 3422319.986, 423...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>Guadalquivir</td>\n",
+       "      <td>5.997583e+05</td>\n",
+       "      <td>MULTILINESTRING ((2859329.283 1682737.074, 286...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "           NAME    Shape_Leng  \\\n",
+       "0        Danube  2.770357e+06   \n",
+       "1         Douro  8.162452e+05   \n",
+       "2          Ebro  8.269909e+05   \n",
+       "3          Elbe  1.087288e+06   \n",
+       "4  Guadalquivir  5.997583e+05   \n",
+       "\n",
+       "                                            geometry  \n",
+       "0  MULTILINESTRING ((4185683.29 2775788.04, 41861...  \n",
+       "1  MULTILINESTRING ((2764963.81 2199037.624, 2766...  \n",
+       "2  MULTILINESTRING ((3178371.814 2315100.781, 317...  \n",
+       "3  MULTILINESTRING ((4235352.373 3422319.986, 423...  \n",
+       "4  MULTILINESTRING ((2859329.283 1682737.074, 286...  "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gpd_rivers_org = gpd.read_file(\n",
+    "    \"https://www.eea.europa.eu/data-and-maps/data/wise-large-rivers-and-large-lakes/\" + \\\n",
+    "    \"zipped-shapefile-with-wise-large-rivers-vector-line/zipped-shapefile-with-wise-large-rivers-vector-line/\" + \\\n",
+    "    \"at_download/file/wise_large_rivers.zip\"\n",
+    ")\n",
+    "gpd_rivers_org.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f8db1757-f80d-4cc3-b650-83109eecc390",
+   "metadata": {},
+   "source": [
+    "Convert to coordinate system / reference system"
```

Comment:
```suggestion
    "Convert to geographic coordinate reference system (longitude/latitude)"
```

Reply:
Improved formulation in commit  https://github.com/GenericMappingTools/agu24workshop/pull/4/commits/ff4dbd9fe45973879e153fcf098e708d9cd30eee.